<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusInfor</class>
 <widget class="QDialog" name="StatusInfor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StatusInfor</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Node Status Information</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1001</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QTableView" name="GeneralTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>591</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="Channels">
    <attribute name="title">
     <string>Channels</string>
    </attribute>
    <widget class="QTableView" name="ChannelTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>591</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="Faces">
    <attribute name="title">
     <string>Faces</string>
    </attribute>
    <widget class="QTableView" name="FaceTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>591</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="FIB">
    <attribute name="title">
     <string>FIB</string>
    </attribute>
    <widget class="QTableView" name="FIBTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>591</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="RIB">
    <attribute name="title">
     <string>RIB</string>
    </attribute>
    <widget class="QTableView" name="RIBTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>591</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="ContentStore">
    <attribute name="title">
     <string>Content Store</string>
    </attribute>
    <widget class="QTableView" name="CSTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>591</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Strategies</string>
    </attribute>
    <widget class="QTableView" name="StrategyTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>591</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>300</number>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
